timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/BatchPaxosAcceptorRpcClient.java

```diff
@@ -114,19 +119,19 @@ AcceptorCacheDigest latestSequencesPreparedOrAccepted(
      * return the sequence number for the latest proposal.
      * <p>
      * If the {@code cacheKey} provided is invalid (expired or never issued) a {@code 404 Not Found} is
-     * returned. The caller should call {@link BatchPaxosAcceptor#latestSequencesPreparedOrAccepted} to get the desired
-     * sequences.
+     * returned with the above conjure error {@link BatchPaxosAcceptorRpcClient#CACHE_KEY_NOT_FOUND}. The caller should
+     * call {@link BatchPaxosAcceptor#latestSequencesPreparedOrAccepted} to get the desired sequences.
      * <p>
      * If a valid {@code cacheKey} is provided, it will return all unseen sequences from when that {@code cacheKey} was
      * issued. If the server has not prepared or accepted any sequences past that point, it will return
-     * {@code 204 No Content}.
+     * {@code 204 No Content} which translates to {@link Optional#empty}.
      * <p>
      * In addition to the updates, a new {@code cacheKey} is provided to use on the next invocation of this method to
      * minimise on payload size as this method is on the hot path.
      *
      * @param paxosUseCase whether this is a timestamp paxos or leader paxos batch call
      * @return {@code 204 No Content} if there is no update, a digest containing updates plus a new cache key, or a
-     * {@code 412 Precondition Failed} if the cache key is not valid.
+     * {@code 404 Precondition Failed} if the cache key is not valid.
```

Comment:
good catch. The message should be `Not Found` I think (but yeah, it is a 404, not a 412)

Reply:
(ping on this)

Reply:
oop, sorry I misinterpreted, fixed now.